quickfixj-core/src/test/java/quickfix/MockSystemTimeSource.java

```diff
@@ -40,7 +40,7 @@ public void setSystemTimes(long[] times) {
         systemTimes = times;
     }
 
-    private void setSystemTimes(long time) {
+    public void setSystemTimes(long time) {
```

Comment:
Just a nitpick: could we remove the access modifier to enable access from within the package?